content/guides/public-transit-equity.mdx

```diff
@@ -415,15 +415,16 @@ Note: Table 2  lists a number of data sources relevant to transit analysis. Howe
 * **[tidytransit](https://r-transit.github.io/tidytransit/)**: Works with GTFS data for transit analysis.
 * **[r5r](https://cran.r-project.org/web/packages/r5r/vignettes/r5r.html)**: Performs routing and accessibility analysis on multimodal transport networks.
 
-<div style={{padding: "1em 1.5em", display:"flex", flexDirection:"column", borderRadius:"1em", backgroundColor:"#ffe5c4"}}>
-  <h4 style={{marginTop:0, marginBottom:".5em"}}>About the Author</h4>
-  <div style={{display:"flex"}}>
-    <div style={{minWidth:"160px"}}>
-      <img className="rounded-full border-4 border-solid border-salmonpink" src="/images/headshot-mg.png" />
-    </div>
-    <p style={{marginLeft:"2em", marginTop:"0"}}><strong>Mandela Gadri</strong> is a Ph.D. student at <strong>University of Illinois at Urbana-Champaign (UIUC)</strong> with an MS in community and regional planning from Iowa State University and a BS in urban planning from Kwame Nkrumah University of Science and Technology, Ghana. He also has a certificate in GIS and worked as a planner with the city of Cedar Rapids, Iowa prior to joining UIUC. Mandela is interested in GIS and transport poverty.</p>
-  </div>
-</div>
+
+<div class='sdoh-panel' style='background-color:#ffe5c4' style={{ backgroundColor: '#ffe5c4' }}>
+<h4 class='sdoh-panel-header'>About the Author</h4>
+<div class='sdoh-panel-content'>
+<img class='sdoh-panel-image' src='/images/headshot-mg.png' />
+<div class='sdoh-panel-body'>
+<span class='sdoh-panel-markdown'>**Mandela Gadri** is a Ph.D. student at **University of Illinois at Urbana-Champaign (UIUC)** with an MS in community and regional planning from Iowa State University and a BS in urban planning from Kwame Nkrumah University of Science and Technology, Ghana. He also has a certificate in GIS and worked as a planner with the city of Cedar Rapids, Iowa prior to joining UIUC. Mandela is interested in GIS and transport poverty.</span>
+<br/>
+<a class='sdoh-panel-link' href='#'> </a>
```

Comment:
Since we don't have a link yet here, temporary workaround was to add a single whitespace character as the Link Text :+1:

The initial spec for this component implied that there would always be an external link, but I can look into making this optional for future cases if we think it might be helpful